thumbor/conf/thumbor.conf.tpl

```diff
@@ -234,15 +234,23 @@ MAX_ID_LENGTH = {{ MAX_ID_LENGTH | default(32) }}
 
 ## Host to send statsd instrumentation to
 ## Defaults to: None
-STATSD_HOST = {{ STATSD_HOST | default(None) }}
+{% if STATSD_HOST is defined %}
+STATSD_HOST = '{{ STATSD_HOST }}'
+{% else %}
+STATSD_HOST = None
+{% endif %}
 
 ## Port to send statsd instrumentation to
 ## Defaults to: 8125
 STATSD_PORT = {{ STATSD_PORT | default(8125) }}
 
 ## Prefix for statsd
 ## Defaults to: None
-STATSD_PREFIX = {{ STATSD_PREFIX | default(None) }}
+{% if STATSD_PREFIX is defined %}
+STATSD_PREFIX = '{{ STATSD_PREFIX }}'
+{% else %}
```

Comment:
I don't think `else` is strictly required though, if the thumbor default is `None` anyway? This is what I did before with `FILTERS` ... If there's no value, we simply fall back to thumbor's default, which I think make sense more than hard coding a default value here.